drivers/spi/spi_silabs_eusart.c

```diff
@@ -152,6 +232,326 @@ static int spi_silabs_eusart_configure(const struct device *dev, const struct sp
 	return 0;
 }
 
+static inline void spi_silabs_eusart_pm_policy_get(const struct device *dev)
+{
+	pm_policy_state_lock_get(PM_STATE_SUSPEND_TO_IDLE, PM_ALL_SUBSTATES);
+	pm_policy_state_lock_get(PM_STATE_STANDBY, PM_ALL_SUBSTATES);
+}
+
+static inline void spi_silabs_eusart_pm_policy_put(const struct device *dev)
+{
+	pm_policy_state_lock_put(PM_STATE_SUSPEND_TO_IDLE, PM_ALL_SUBSTATES);
+	pm_policy_state_lock_put(PM_STATE_STANDBY, PM_ALL_SUBSTATES);
+}
+
+static int spi_silabs_eusart_power_management_pm_action(const struct device *dev,
```

Comment:
This is still an unnecessarily long name. Remove `power_management` from the name, `pm` is already there.